db/schema.rb

```diff
@@ -780,7 +768,6 @@
     t.index ["id", "imported_at"], name: "index_youtube_profiles_on_id_and_imported_at"
   end
 
-  add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
```

Comment:
@danswick This seems like an accidental/unrelated change.  Try the following:

1. Rollback this file:
`git checkout main -- db/schema.rb`
2. `rails db:drop`
3. `rails db:create`
4. `rails db:setup`
5.`rails db:migrate`

And then check the changes in the new `db/schema.rb`.  Is this change there?  (For me it isn't).

Reply:
Thank you for catching that! I'm not sure how that change even happened, but it should be rolled back now.